src/components/App/index.jsx

```diff
@@ -55,64 +62,67 @@ const AppWrapper = () => {
   }, [config]);
 
   return (
-    <AppProvider store={store}>
-      <Helmet
-        titleTemplate="%s - edX Admin Portal"
-        defaultTitle="edX Admin Portal"
-      />
-      {isMaintenanceAlertOpen && (
-        <SystemWideWarningBanner>
-          {config.MAINTENANCE_ALERT_MESSAGE}
-        </SystemWideWarningBanner>
-      )}
-      <Header />
-      <Switch>
-        <AuthenticatedPageRoute
-          path="/enterprises"
-          render={(routerProps) => <EnterpriseIndexPage {...routerProps} />}
-          authenticatedAPIClient={apiClient}
-          redirect={`${process.env.BASE_URL}/enterprises`}
-        />
-        <PageRoute
-          exact
-          path="/:enterpriseSlug/admin/register"
-          component={AdminRegisterPage}
-        />
-        <PageRoute
-          exact
-          path="/:enterpriseSlug/admin/register/activate"
-          component={UserActivationPage}
-        />
-        <PageRoute
-          path="/:enterpriseSlug"
-          authenticatedAPIClient={apiClient}
-          redirect={process.env.BASE_URL}
-          render={({
-            match: {
-              url: baseUrl,
-            },
-          }) => (
-            <Switch>
-              <Route
-                path="/:enterpriseSlug/admin/:enterpriseAppPage"
-                component={AuthenticatedEnterpriseApp}
-              />
-              <Redirect
-                to={`${baseUrl}/admin/${ROUTE_NAMES.learners}`}
-              />
-            </Switch>
-          )}
-        />
-        <AuthenticatedPageRoute
-          path="/"
-          render={(routerProps) => <EnterpriseIndexPage {...routerProps} />}
-          authenticatedAPIClient={apiClient}
-          redirect={process.env.BASE_URL}
+    <QueryClientProvider client={queryClient}>
+      <ReactQueryDevtools />
```

Comment:
[context] Wraps the existing `App` component with `QueryClientProvider` and includes `ReactQueryDevtools`. Everything else is unchanged.